### ISSUE TITLE
Nick getting white on the tablist.

### ISSUE THREAD

**Issue by Dcore18** [sgtcaze/NametagEdit#297]
# Bug Report

**Type /ver NametagEdit and post the output.**
Lastest version in build #38

**What version of Bukkit/Spigot/PaperSpigot are you using? Type /ver**
Lastest spigot version

**What plugins are you using? Type /plugins**

**Please explain your issue. How do you trigger it?**
Every time you change the world, alternating between them. Your nick will be white on the tab.

**Are there any errors in the console? Please use: https://pastebin.com**
https://gyazo.com/efcf2dfb90a4f27db5aab73e69681016

**Comment by JustOneMoreBlock** [comment]
Latest Spigot Version: 1.12.x or 1.13.x?

**Comment by Dcore18** [comment]
1.12.x

**Comment by Dcore18** [comment]
Progress ?

**Comment by danjono** [comment]
I cannot replicate this for you @Dcore18. I've tried a 1.12.2 server with default config and the latest dev version of Nametagedit and swapped worlds with Multiverse and my nametag stays the same. 

What plugin are you using to do your tablist and scoreboard? Also in the config.yml do you have RefreshTagOnWorldChangeturned on/off?

**Comment by Dcore18** [comment]
I use the nametag for months on 1.8, I use the same settings on the 1.12. Something changed because the 1.8 didn't have that problem.

**Comment by danjono** [comment]
Could you answer my questions please. You will not be able to get any support if this issue cannot be replicated by someone else. It may just be easier to give a screenshot of all your plugins with /pl as it could also be something to do with your permissions plugin.

**Comment by Dcore18** [comment]
My dear understand, I use the plugin for months never had problems. In the update stopped working, permissions are correct even because I am a director, when changing the world your name turns white, only yours.

**Comment by danjono** [comment]
Sorry, I can't help you then if you don't wish to provide the additional information needed to help diagnose this problem further.

@JustOneMoreBlock there's not much point in keeping this issue open. I've experienced something similar before with BungeeTabListPlus when I tried to have per world tablists, it's likely something similar but without any additional informational from this person there's not much can be done to help diagnose.
